low-checker.user.js

```diff
@@ -73,7 +75,13 @@
         oneSentencePerLineValidator: function (line) {
             const parts = line.split('.');
             return parts.length > 2 ? "Suspected: more than one sentence in one line." : "";
-        }
+        },
+        nonAsciiValidator: function (line) {
+            const nonAsciiSymbols = line.match(nonAsciiRegexp);
+            return nonAsciiSymbols?.length
+                ? `${nonAsciiSymbols.length} incorrect symbols were detected: ${nonAsciiSymbols.slice(0, 5).join(", ")}${nonAsciiSymbols.length > 5 ? ", etc." : ""}`
+                : "";
```

Comment:
Use `if` construction instead. And use the following trick:

```
nonAsciiSymbols[5] = "etc."
...
nonAsciiSymbols.filter(el => el !== '').join(", ")
```

It will make code more clean and readable. :)